modules/sno-du-configuring-the-operators.adoc

```diff
@@ -6,7 +6,7 @@
 [id="sno-du-configuring-the-operators_{context}"]
 = Creating OperatorGroups for Operators
 
-This configuration is provided to enable addition of the Operators needed to configure the platform post-installation. It adds the `Namespace` and `OperatorGroup` objects for the Local Storage Operator, Logging Operator, Performance Addon Operator, PTP Operator, and SRIOV Network Operator.
+This configuration is provided to enable addition of the Operators needed to configure the platform post-installation. It adds the `Namespace` and `OperatorGroup` objects for the Local Storage Operator, Logging Operator, PTP Operator, and SRIOV Network Operator.
```

Comment:
Just wanted to confirm this: Is the 'Performance Addon Operator' to be removed from the list, or do we need to add  the new term 'Node Tuning Operator' here?

Reply:
Performance addons opeartor will no longer be an operator to be installed (day-1)  since it has moved as a controller encapsulated under Node Tuning operator.  
So in terms of installation when deploying openshift you already get these capabilities as day-0